docs/content/en/enterprise.html

```diff
@@ -0,0 +1,72 @@
++++
+title = "Tetragon - eBPF-based Security Observability and Runtime Enforcement"
+description = "Learn how enterprise support and training for Tetragon provides essential resources for reliable, large-scale deployments."
+linkTitle = "Tetragon - eBPF-based Security Observability and Runtime Enforcement"
+kind = "home"
++++
+
+<div class="home">
+
+	<div class="hero hero--inner">
+		<div class="container">
+			<div class="hero__intro">
+				<h1 class="title title--xl">
+					Tetragon Enterprise Distributions & Training
+				</h1>
+				<p>
+					Initially created by Isovalent, Tetragon is a CNCF-backed project dedicated to real-time
+					observability and security for Kubernetes. Certified partners provide enterprise distributions,
+					training, and&nbsp;commercial support for Tetragon.
```

Comment:
Can we change "Certified partners provide enterprise distributions, training, and commercial support for Tetragon" to "The listed partners offer enterprise distributions, training, and commercial support for Tetragon", which is the wording that Cilium uses (https://cilium.io/enterprise/) ?

Reply:
Done ✅